include/graphene-box2d.h

```diff
@@ -0,0 +1,122 @@
+                                                                     const graphene_point_t *point,
+                                                                     graphene_box2d_t       *res);
+GRAPHENE_AVAILABLE_IN_1_12
+void                      graphene_box2d_expand_vec2                (const graphene_box2d_t *box,
+                                                                     const graphene_vec2_t  *vec,
+                                                                     graphene_box2d_t       *res);
+GRAPHENE_AVAILABLE_IN_1_12
+void                      graphene_box2d_expand_scalar              (const graphene_box2d_t *box,
+                                                                     float                   scalar,
+                                                                     graphene_box2d_t       *res);
+GRAPHENE_AVAILABLE_IN_1_12
+void                      graphene_box2d_union                      (const graphene_box2d_t *a,
+                                                                     const graphene_box2d_t *b,
+                                                                     graphene_box2d_t       *res);
+GRAPHENE_AVAILABLE_IN_1_12
+bool                      graphene_box2d_intersection               (const graphene_box2d_t *a,
+                                                                     const graphene_box2d_t *b,
+                                                                     graphene_box2d_t       *res);
+
+GRAPHENE_AVAILABLE_IN_1_12
+float                     graphene_box2d_get_width                  (const graphene_box2d_t *box);
+GRAPHENE_AVAILABLE_IN_1_12
+float                     graphene_box2d_get_height                 (const graphene_box2d_t *box);
+GRAPHENE_AVAILABLE_IN_1_12
+void                      graphene_box2d_get_size                   (const graphene_box2d_t *box,
+                                                                     graphene_vec2_t        *size);
+GRAPHENE_AVAILABLE_IN_1_12
+void                      graphene_box2d_get_center                 (const graphene_box2d_t *box,
+                                                                     graphene_point_t       *center);
+GRAPHENE_AVAILABLE_IN_1_12
+void                      graphene_box2d_get_min                    (const graphene_box2d_t *box,
+                                                                     graphene_point_t       *min);
+GRAPHENE_AVAILABLE_IN_1_12
+void                      graphene_box2d_get_max                    (const graphene_box2d_t *box,
+                                                                     graphene_point_t       *max);
+GRAPHENE_AVAILABLE_IN_1_12
+void                      graphene_box2d_get_vertices               (const graphene_box2d_t *box,
+                                                                     graphene_vec2_t         vertices[]);
```

Comment:
A `graphene_box2d_to_float (box, float[4])` would come in handy here for uploading to GLSL.